src/components/include/jupyterlite/BaseJupyterLiteComponent.tsx

```diff
@@ -97,6 +97,7 @@ class BaseJupyterLiteSessionComponent<P = never, S = never> extends React.Compon
     render() {
         return (
             <JupyterLiteSession
+                originURL="https://deploy-preview-56--mat3ra-jupyterlite.netlify.app"
```

Comment:
Maybe get this from a settings file with the following logic: if `development` profile, use `https://deploy-preview-{JupyterLitePullRequestNumber}--mat3ra-jupyterlite.netlify.app`. So that you only need to update:

`isDevelopment: true, JupyterLitePullRequestNumber: 56`